### PR TITLE
linting fixes

### PR DESCRIPTION
**Pull request by jgunnCO**
* Enable prettier as part of linting
* Use typescript parser and lint rules on typescript files
* Address the large number of errors produced by fixing the above
* Unify lint config and dependencies across repo - linting now all done from root dir

### CODE REVIEW

File: packages/applicant/src/components/partials/Header.tsx [hunk 1]
```diff
@@ -1,199 +0,0 @@
...
-              will help us to improve it.
-            </span>
-          </p>
-        </div>
-      </div>
-    </>
-  );
-};
-
-export default Header;
```
Comment (john-tco) [inline packages/applicant/src/components/partials/Header.tsx:199]: This component is still being imported in Layout.tsx

File: packages/applicant/src/pages/api/isNewApplicant.page.tsx [hunk 2]
```diff
@@ -1,16 +0,0 @@
...
-  res: NextApiResponse
-) {
-  const grantApplicantService = GrantApplicantService.getInstance();
-  const doesApplicantExist: Boolean =
-    await grantApplicantService.doesApplicantExist(getJwtFromCookies(req));
-  if (!doesApplicantExist) {
-    await grantApplicantService.createAnApplicant(getJwtFromCookies(req));
-  }
-  res.redirect(process.env.APPLICANT_FRONTEND_URL);
-}
```
Comment (john-tco) [inline packages/applicant/src/pages/api/isNewApplicant.page.tsx:16]: I can still see this handler used in routes.tsx - do we still need it?

File: packages/applicant/src/services/GrantSchemeService.ts [hunk 3]
```diff
@@ -1,32 +0,0 @@
...
-    grantSchemeId: String,
-    jwt: string
-  ): Promise<GrantScheme> {
-    const { data } = await axios.get<GrantScheme>(
-      `${this.BACKEND_HOST}/grant-schemes/${grantSchemeId}`,
-      axiosConfig(jwt)
-    );
-    return data;
-  }
-}
```
Comment (john-tco) [inline packages/applicant/src/services/GrantSchemeService.ts:32]: why have you deleted this :sob:

File: packages/applicant/src/components/partials/Header.tsx (hunk 1 above)
Comment (jgunnCO) [reply packages/applicant/src/components/partials/Header.tsx:199]: I actually have no idea where all these file deletions are coming from, I certainly didn't delete them manually.

Comment (jgunnCO) [reply packages/applicant/src/components/partials/Header.tsx]: Deleted files resotred

**Review by john-tco — APPROVED** [review]

**Pull request merged**